knowledge-base/common-maximum-call-stack-exceeded.md

```diff
@@ -0,0 +1,70 @@
+---
+title: JSException Maximum Call Stack Size Exceeded
+description: Learn what causes the Maximum call stack size exceeded error and how to fix it.
+type: troubleshooting
+page_title: How to Fix JSException Maximum Call Stack Size Exceeded
+slug: common-kb-maximum-call-stack-exceeded
+tags: telerik, blazor, troubleshooting, javascript, js-error, exception
+ticketid: 1641599, 1638959, 1639186, 1638288, 1642561, 1645266, 1641021
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This knowledge base article deals with the `Maximum call stack size exceeded` error.
+
+## Error Message
+
+>warning Microsoft.JSInterop.JSException: Maximum call stack size exceeded
+>
+> RangeError: Maximum call stack size exceeded
+
+## Cause
+
+The possible causes for the `Maximum call stack size exceeded` error are:
+
+* Using an old version of Telerik UI for Blazor (4.5.0 and older) in .NET 8.
```

Comment:
Don't use "old" and "new" when [referencing product versions](https://docs.telerik.com/style-guide/vocabulary#list-of-terms). Use "earlier" and "later" instead.
```suggestion
* Using an old version of Telerik UI for Blazor (4.5.0 and earlier) in .NET 8.
```